control-plane/grpc/proto/v1/node/node.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+import "v1/misc/common.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/wrappers.proto";
+
+package v1.node;
+
+// Node information
+message Node {
+  // Node identification
+  string node_id = 1;
+  // Specification of the node.
+  optional NodeSpec spec = 2;
+  // Runtime state of the node.
+  optional NodeState state = 3;
+}
+
+message NodeSpec {
+  // Node identification
+  string node_id = 1;
+  // Endpoint of the mayastor instance (gRPC)
+  string endpoint = 2;
+  // Node labels.
+  common.StringMapValue labels = 3;
+}
+
+message NodeState {
+  // id of the mayastor instance
+  string node_id = 1;
+  // grpc_endpoint of the mayastor instance
+  string endpoint = 2;
```

Comment:
Do we need the endpoint in the state? We already have it in the spec. Or do we require it in order to update the spec (in case the endpoint changes)?

Reply:
:+1: